<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTIN204528" failures="0" tests="5" name="testCases.TC001_BeCognizant" time="0.031" errors="1" timestamp="2024-01-16T11:16:57 IST" skipped="4">
  <testcase classname="testCases.TC001_BeCognizant" name="setup" time="0.031">
    <error message=".\src	estesources\config.properties (The filename, directory name, or volume label syntax is incorrect)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: .\srcestesources\config.properties (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at java.base/java.io.FileReader.<init>(FileReader.java:60)
at testBase.BaseClass.setup(BaseClass.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)
at org.testng.TestRunner.beforeRun(TestRunner.java:652)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="testCases.TC001_BeCognizant" name="beCognizantTextValidation" time="0.000">
    <skipped/>
  </testcase> <!-- beCognizantTextValidation -->
  <testcase classname="testCases.TC001_BeCognizant" name="profileValidation" time="0.000">
    <skipped/>
  </testcase> <!-- profileValidation -->
  <testcase classname="testCases.TC001_BeCognizant" name="oneCognizantTextValidation" time="0.000">
    <skipped/>
  </testcase> <!-- oneCognizantTextValidation -->
  <testcase classname="testCases.TC001_BeCognizant" name="navigateOneCognizant" time="0.000">
    <skipped/>
  </testcase> <!-- navigateOneCognizant -->
</testsuite> <!-- testCases.TC001_BeCognizant -->
